app/commands/source/add_value_command.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+module Source
+  class AddValueCommand < ApplicationCommand
+    usage do
+      no_command
+    end
+
+    keyword :name do
+      desc "Name for the new value"
+      required
+    end
+
+    keyword :attribute_friendly_id do
+      desc "Friendly ID of the primary attribute to add the value to"
+      required
+    end
+
+    def execute
+      frame("Adding new value") do
+        ActiveRecord::Base.transaction do
+          setup!
+          create_value!
+          create_attributes_value!
```

Comment:
Can this be pushed inside of value creation as well? I feel like we did something similar in the parser?